libfreerdp/core/freerdp.c

```diff
@@ -1131,6 +1131,24 @@ BOOL freerdp_nla_revert_to_self(rdpContext* context)
 	return nla_revert_to_self(nla);
 }
 
+UINT32 freerdp_get_nla_sspi_error(rdpContext* context)
+{
+	rdpNla* nla;
+
+	if (!context)
```

Comment:
better `WINPR_ASSERT` these, at least `context`, `context->rdp` and `context->rdp->transport` must not be `NULL`

Reply:
fixed